docs/faq.md

```diff
@@ -22,6 +23,70 @@ If a question you have is not answered below, please [submit an issue](/../../is
 
 See [rules_docker](https://github.com/bazelbuild/rules_docker) for a similar existing container image build tool for the [Bazel build system](https://github.com/bazelbuild/bazel). The tool can build images for languages such as Python, NodeJS, Java, Scala, Groovy, C, Go, Rust, and D.
 
+### How do I run the image I built?
+
+If you built your image directly to the Docker daemon using `jib:dockerBuild` (Maven) or `jibDockerBuild` (Gradle), you simply need to use `docker run <image name>`.
+
+If you built your image to a registry using `jib:build` (Maven) or `jib` (Gradle), you will need to pull the image using `docker pull <image name>` before using `docker run`.
+
+To run your image on Kubernetes, see [steps 4-6 of the Kubernetes Engine deployment tutorial](https://cloud.google.com/kubernetes-engine/docs/tutorials/hello-app#step_4_create_a_container_cluster).
```

Comment:
I think we can provide the kubectl command here like in https://github.com/coollog/micronaut-jib#with-kubernetes